CP04OSSM/CP04OSSM_R00005_ingest.csv

```diff
@@ -1,43 +1,43 @@
+mi.dataset.driver.pco2w_abc.pco2w_abc_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00005/instruments/dcl35/PCO2W_sn*/SAMI*.txt,CP04OSSM-MFD35-05-PCO2WB000,recovered_inst,Available,
+mi.dataset.driver.phsen_abcdef.dcl.phsen_abcdef_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00005/cg_data/dcl35/phsen2/*.phsen2.log,CP04OSSM-MFD35-06-PHSEND000,recovered_host,Available,
+mi.dataset.driver.phsen_abcdef.phsen_abcdef_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00005/instruments/dcl35/PHSEN_snP*/SAMI*.txt,CP04OSSM-MFD35-06-PHSEND000,recovered_inst,Available,
+mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00005/cg_data/dcl37/syslog/*.syslog.log,CP04OSSM-MFD37-00-DCLENG000,recovered_host,Available,
+mi.dataset.driver.optaa_dj.dcl.optaa_dj_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00005/cg_data/dcl37/optaa2/*.optaa2.log,CP04OSSM-MFD37-01-OPTAAD000,recovered_host,Available,
+mi.dataset.driver.ctdbp_cdef.dcl.ctdbp_cdef_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00005/cg_data/dcl37/ctdbp2/*.ctdbp2.log,CP04OSSM-MFD37-03-CTDBPE000,recovered_host,Available,
+mi.dataset.driver.ctdbp_cdef.ctdbp_cdef_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00005/instruments/dcl37/CTDBP_sn*/SBE*.hex,CP04OSSM-MFD37-03-CTDBPE000,recovered_inst,Available,
+mi.dataset.driver.dosta_abcdjm.dcl.dosta_abcdjm_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00005/cg_data/dcl37/dosta2/*dosta2.log,CP04OSSM-MFD37-04-DOSTAD000,recovered_host,Available,
+mi.dataset.driver.zplsc_c.dcl.zplsc_c_dcl_telemetered_driver,/omc_data/whoi/OMC/CP04OSSM/R00005/cg_data/dcl37/zplsc/*.zplsc.log,CP04OSSM-MFD37-07-ZPLSCC000,recovered_host,Available,
+mi.dataset.driver.cg_cpm_eng.cpm.cg_cpm_eng_cpm_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00005/cg_data/cpm2/syslog/cpm_status*.txt,CP04OSSM-RIC21-00-CPMENG000,recovered_host,Available,
+mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00005/cg_data/dcl26/syslog/*.syslog.log,CP04OSSM-RID26-00-DCLENG000,recovered_host,Available,
+mi.dataset.driver.velpt_ab.dcl.velpt_ab_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00005/cg_data/dcl26/velpt1/*.velpt1.log,CP04OSSM-RID26-04-VELPTA000,recovered_host,Available,
+mi.dataset.driver.velpt_ab.velpt_ab_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00005/instruments/dcl26/VELPT_snAQD*/*.aqd,CP04OSSM-RID26-04-VELPTA000,recovered_inst,Available,
+mi.dataset.driver.phsen_abcdef.dcl.phsen_abcdef_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00005/cg_data/dcl26/phsen1/*.phsen1.log,CP04OSSM-RID26-06-PHSEND000,recovered_host,Available,
+mi.dataset.driver.phsen_abcdef.phsen_abcdef_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00005/instruments/dcl26/PHSEN_snP*/SAMI*.txt,CP04OSSM-RID26-06-PHSEND000,recovered_inst,Available,

```

Comment:
data are available

Reply:
Suggest moving the bottom ZPLSC line up to the other one.
ADCP PD8 recovered host data should not be ingested.